### DISCUSSION TITLE
Hunleff Helper-Like sounds in BaBa monkey room based on typed keyphrases

### DISCUSSION THREAD

**Discussion by Sebas613** [runelite/runelite#17872] [Ideas]
Hi,

One of the more annoying things about the monkey room when you're used to solo's, is remembering to look at chat for the P / V / Dd callouts.

It would be a huge help if a plugin could be made, that plays the key sounds on certain keywords, like the 3..2...1... Mage.

*p* / *ill* = Pillars sound effect
*v* *f* = vents sound effect
*d* = dd sound effect

This plugin could only play in the monkey room itself, similarly to how the hunleff helper has an option for the overlay to only appear inside the hunleff room.

If possible, I'd love this. Thanks :)

**Comment by cdfisher** [comment]
This would (probably, I guess?) be against Jagex's third-party client guidelines. Hunllef Helper is allowed because it just runs on a manually triggered timer.

**Comment by Sebas613** [comment]
It would just make team-play a bit more relaxed for those used to always running team raids with discord voice comms. It is more so an even playing field for those who don't use "more" than anything else. Would love it if they could ask.

**Comment by Sebas613** [comment]
Though, if the "manually started timer" thing is the only relative thing: A timer that just audibly says "check chat for callout" would be less nice but do the same, since the assignment of the events is by itself also on a timer.

**Comment by harbingerofme** [comment]
Try: 
https://runelite.net/plugin-hub/show/tts and https://runelite.net/plugin-hub/show/naturalspeech

**Reply by Sebas613** [reply]
holy nice, just gotta remember to turn it on/off during that room but will do for now I guess. Thanks!

**Reply by Sebas613** [reply]
noticed the 2nd one even has an option to spell out "vents" for v. huuuuuGE!
